ams/server/src/main/java/com/netease/arctic/server/table/TableService.java

```diff
@@ -84,6 +84,8 @@ public interface TableService extends CatalogService, TableManager {
    */
   List<TableIDWithFormat> listTables(String catalogName, String dbName);
 
+  TableRuntime getRuntime(TableIdentifier tableIdentifier);
+
```

Comment:
I'm not sure if the interface matches the design of the `TableService`.

Reply:
We already have a interface `com.netease.arctic.server.table.TableManager#getRuntime`, there is no need to add a new one.

Reply:
I have reviewed com.netease.arctic.server.table.TableManager#getRuntime, which confused me is that we can not build ServerTableIdentifier only with catalog/db/tab from frontend request params.  
We can not get TableRuntime from tableRuntimeMap (private final Map<ServerTableIdentifier, TableRuntime> tableRuntimeMap ), because ServerTableIdentifier's equals method also contains tableId except catalog/db/tab.

Reply:
Yes, `ServerTableIdentifier` contains a field `tableId` to distinguish tables with the same catalog/db/table, perhaps it is a table been dropped and then recreated.

I think we can add an interface(method) to `com.netease.arctic.server.table.TableService` that can get `ServerTableIdentifier` from `TableIdentifier`, and this method has already been implemented